signbank/dictionary/tests.py

```diff
@@ -615,17 +615,12 @@ def test_Export_csv(self):
         assign_perm('dictionary.export_csv', self.user)
         print('User has permmission to export csv.')
 
-        # set the language for glosssearch field name
-        default_language = self.test_dataset.default_language
-        gloss_search_field_prefix = "glosssearch_"
-        glosssearch_field_name = gloss_search_field_prefix + default_language.language_code_2char
-
-        response = client.get('/signs/search/', {"search_type": "sign", glosssearch_field_name : "wesseltest6", "format": "CSV"})
-
+        response = client.get('/signs/search/',
+                              {'format': 'CSV'}, follow=True)
```

Comment:
When entering a query and pressing enter, it directly downloads the dictionary CSV instead of showing the signs for that query